.editorconfig

```diff
@@ -10,6 +10,9 @@ insert_final_newline = true
 indent_style = tabs
 indent_size = 4
 
-[{package.json,bower.json,*.md}]
+[*.csv]
+trim_trailing_whitespace = false
+
+[{bower.json,package.json,package-lock.json,*kml,*.md}]
```

Comment:
We might be able to just say `*.json` unless there is some we want to keep. I don't mind the tabs in JSON since it's smaller on the wire.